<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MolecularPropertiesDialog</class>
 <widget class="QDialog" name="MolecularPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Molecule Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="molWeightLabel">
       <property name="text">
        <string>Molecular Weight (g/mol):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="formulaLabel">
       <property name="text">
        <string>Chemical Formula:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="atomsLabel">
       <property name="text">
        <string>Number of Atoms:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="bondsLabel">
       <property name="text">
        <string>Number of Bonds:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="residuesLabel">
       <property name="text">
        <string>Number of Residues:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="energyLabel">
       <property name="text">
        <string>Energy (kJ/mol):</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="dipoleLabel">
       <property name="text">
        <string>Dipole Moment (D):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="molecularWeightLine">
       <property name="readOnly" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="formulaLine">
       <property name="readOnly" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="energyLine">
       <property name="readOnly" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="dipoleMomentLine">
       <property name="readOnly" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="atomsLine">
       <property name="readOnly" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="bondsLine">
       <property name="readOnly" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="residuesLine">
       <property name="readOnly" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MolecularPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
